orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -255,14 +256,14 @@ private static void setDbAdaptors(MySQLAdaptor dbAdaptor)
 		OrthologousEntityGenerator.setAdaptor(dbAdaptor);
 		EWASInferrer.setAdaptor(dbAdaptor);
 		PathwaysInferrer.setAdaptor(dbAdaptor);
-		
+
 	}
 
 	// Read the species-specific orthopair 'mapping' file, and create a HashMap with the contents
 	private static Map<String, String[]> readHomologueMappingFile(String toSpecies, String fromSpecies, String pathToOrthopairs) throws IOException
 	{
 		String orthopairsFileName = fromSpecies + "_" + toSpecies + "_mapping.txt";
-		String orthopairsFilePath = pathToOrthopairs + orthopairsFileName;
+		String orthopairsFilePath = pathToOrthopairs + "/" + orthopairsFileName;
```

Comment:
Whenever possible, try to use `Path.get()` to avoid hard-coding the file separator as "/"